_includes/world/2024/sections/speakers.html

```diff
@@ -2,28 +2,30 @@
 {% assign keynote_speakers = sorted_speakers | where: 'keynote', true %}
 {% assign non_keynote_speakers = sorted_speakers | where_exp: "speaker", "speaker.keynote != true" | sort: "first_name" %}
 
+
 <section class="content-container speaker-cards">
     <div class="headerContainer">
         <div class="textContainer">
             <h3>Keynote Speakers</h3>
             <p>
-                Gain invaluable insights from our four keynote speakers on the present landscape of Rails and where Rails is headed tomorrow and beyond. More speakers announced soon.
+                Gain invaluable insights from our four keynote speakers on the present landscape of Rails and where
+                Rails is headed tomorrow and beyond. More speakers announced soon.
             </p>
         </div>
-    <!-- <div class="buttonsContainer">
+        <div class="buttonsContainer">
```

Comment:
Correct me if I am wrong, but is this adding the arrows back? If so, those are not needed for now. We'll need them when we add more speakers soon.

Reply:
I actually moved this into the css styles - so it's easier to remove when we want too. 

I just set it to `display: none;`

Reply:
> it's easier to remove when we want too.

Meaning when we want to remove all these changes for the 4 speakers, it's all in the one css file